### PR TITLE
Update downstream to be based on rpms

### PR DESCRIPTION
**Pull request by jcantrill**
This PR satisfies https://trello.com/c/0VsVlN5W

cc @adammhaile

### CODE REVIEW

**Comment by jcantrill** [comment]
[(3) Productize upstream curator 5.2 [epic-ois-es5]](https://trello.com/c/0VsVlN5W/676-3-productize-upstream-curator-52-epic-ois-es5)

File: curator/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@
 #oit## reconciliation process.
 #oit## 
 FROM rhel7:7-released
+#oit## Added automatically. empty.repo disables all below specified yum repos, so that OSBS/ODCS can manage repos instead.
+ADD ./.oit/empty.repo /etc/yum.repos.d/
```
Comment (adammhaile) [inline curator/Dockerfile]: This is automatically added by us. Please remove.

File: curator/Dockerfile [hunk 2]
```diff
@@ -51,15 +50,18 @@ WORKDIR ${HOME}
 USER 1001
 CMD ["sh", "run.sh"]
 
+
+RUN rm -f /etc/yum.repos.d/empty.repo  || true
```
Comment (adammhaile) [inline curator/Dockerfile]: Also remove.

**Comment by jcantrill** [comment]
Merging as all tests passed and these changes do not affect the outcome

**Pull request merged**